iothub/device/src/Transport/Amqp/AmqpAuthenticationRefresher.cs

```diff
@@ -98,8 +85,8 @@ private async Task RefreshLoopAsync(DateTime expiry, CancellationToken cancellat
                             ConnectionString,
                             ConnectionString.AmqpEndpoint,
                             Audience,
-                            Audience,
-                            AccessRightsHelper.AccessRightsToStringArray(AccessRights.DeviceConnect),
+                            WebUtility.UrlEncode(Audience),
```

Comment:
@varunpuranik PTAL - I don't think this is right either.

Reply:
Yeah, this looks unnecessary, but I haven't looked at other code paths.